src/main/java/com/solvd/webtest/pages/base/HomePageBase.java

```diff
@@ -0,0 +1,21 @@
+package com.solvd.webtest.pages.base;
+
+import com.zebrunner.carina.utils.R;
+import org.openqa.selenium.WebDriver;
+
+public abstract class HomePageBase extends BasePage {
+    public HomePageBase(WebDriver driver) {
+        super(driver, R.CONFIG.get("homePage.url"));
+    }
+
+    @Override
+    public void open() {
```

Comment:
do you need to override it? If I am not mistaken, you can use it without overriding, you do not change anything in this method so you can just use one from the parent class

Reply:
Ok, this may be unnecessary